packages/http-client-csharp/eng/scripts/Get-CadlRanch-Coverage.ps1

```diff
@@ -78,11 +80,14 @@ foreach ($directory in $directories) {
 
 # test all
 Write-Host "Generating CadlRanch coverage" -ForegroundColor Cyan
-$command  = "dotnet test $cadlRanchCsproj"
-Invoke $command
-# exit if the testing failed
-if ($LASTEXITCODE -ne 0) {
-    exit $LASTEXITCODE
+foreach ($csprojFile in $csprojFiles) {
+    Write-Host "Testing $csprojFile" -ForegroundColor Cyan
+    $command = "dotnet test $csprojFile"
```

Comment:
Can we verify that the coverage report looks right locally?

Reply:
ok - let me check that to ensure doing this in this way and it could be accumulative